### PR TITLE
[blog] Announce CNCF Sandbox

### PR DESCRIPTION
**Pull request by stefanalund**
#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

**Review by freben — APPROVED** [review]
🎉

File: microsite/blog/2020-09-23-backstage-cncf-sandbox.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+authorURL: https://twitter.com/stalund
+---
+
+**TL;DR** The Cloud Native Computing Foundation (CNCF) announced that Backstage can begin incubating as an early stage project in the [CNCF Sandbox](https://www.cncf.io/sandbox-projects/). Released open source in March, the platform is built around an advanced service catalog and is designed to streamline software development from end to end.
+
+![cncf](assets/cncf-sandbox/cncf.png)
+
+<!--truncate-->
+
+Backstage garnered quite a bit of interest from developers and organizations when it was first announced, and community interest continues to grow as plugins and new features are added with the open source community. We released the open source version of Backstage ‘early’. That was intentional. Because even though we’ve been using Backstage internally for years, we wanted the open source version to be developed with input and contributions from the community. And that’s exactly the product that’s going into the [CNCF Sandbox](https://www.cncf.io/sandbox-projects/) today.
```
Comment (benjdlambert) [inline microsite/blog/2020-09-23-backstage-cncf-sandbox.md:13]: TIL `garnerede`

**Review by benjdlambert — APPROVED** [review]

**Review by shmidt-i — APPROVED** [review]
🎉

**Review by soapraj — APPROVED** [review]

**Comment by stefanalund** [comment]
This was a well received pull request 😏

**Pull request merged**